rust/repo/examples/default/output/repository/.github/workflows/ci.yaml

```diff
@@ -39,6 +39,10 @@ jobs:
       pull-requests: write
     steps:
       - uses: actions/checkout@v3
+{%- if cookiecutter.submodules == 'true' %}
+        with:
+          submodules: true
+{%- endif %}
```

Comment:
i'm not sure why it's not replacing these directives in this file, i tried a few variations. it seems to work in `.github/settings.yaml` and other places.

Reply:
Ah, yeah, ugh. So this is happening because we're just copying the github workflows directly into the output, which we wanted to do in order to avoid needing escape github's `{{` variable syntax. I just ran into this on the groups, orgs, and teams PR.